src/flexmeasures_client/client.py

```diff
@@ -409,15 +410,15 @@ async def get_sensor_data(
         entity_address: str,
         resolution: str | timedelta,
     ) -> dict:
```

Comment:
Some parameters are missing: horizon, prior, source. I would add them by declaring them explicitly or by allow passing them as keyword arguments.

Reply:
added **kwargs to the function